README.rst

```diff
@@ -42,4 +42,235 @@ Other code (in ``libcore``, ``lib``, etc.) should not be used by external users
+``codelist_url``            string                  A url that the codelist csv is accessible at, e.g. ``https://raw.githubusercontent.com/open-contracting/standard/1.1/schema/codelists/documentType.csv``
+``codelist_amend_urls``     array[array[string]     Urls to codelist csvs from extensions that amend the codelist. Structure is an array of pairs of ``+`` or ``-`` and the url. e.g. ``[["+", "https://raw.githubusercontent.com/open-contracting-extensions/ocds_tariffs_extension/d9df2969030b0a555c24c7db685262c714b4da24/codelists/+documentType.csv"]]``
+``isopen``                  boolean                 Is this an open codelist?
+``values``                  array*                  Values in the data that were not on the codelist
+``extension_codelist``      boolean                 Was the codelist added by an extension? (Not just modified by it).
+=========================== ======================= ============
+
+additional check object
+^^^^^^^^^^^^^^^^^^^^^^^
+
+=========================== ===================== ==============
+Property (key) name	    Type                  Value
+=========================== ===================== ==============
+``json_location``           string                A JSON Pointer to the data that was problematic e.g. ``releases/0/buyer``
+=========================== ===================== ==============
+
+
+conformance_errors
+^^^^^^^^^^^^^^^^^^
+
+=============================== ======================= =====
+Property (key) name	        Type                    Value
+=============================== ======================= =====
+``ocds_prefixes_bad_format``    array[array[string]]    An array of pairs of the bad ocid value, and the JSON Pointer to it, e.g. ``["MY-ID", "releases/0/ocid"]``
+``ocid_description``            string                  The descriptive text about ocids taken from the schema.
+``ocid_info_url``               string                  The url to the information about identifiers in the reference docs
+=============================== ======================= =====
+
+additional_fields
+^^^^^^^^^^^^^^^^^
+
+============================= ========= ==============
+Property (key) name	      Type      Value
+============================= ========= ==============
+``path``                      string    The path of the parent object, e.g. ``/publisher``
+``field``                     string    The JSON property name, e.g. ``myField``
+``usage_count``               integer   How many times this additional field appears
+============================= ========= ==============
+
+all_additional_fields
+^^^^^^^^^^^^^^^^^^^^^
+
+=================================== =========== ==============
+Property (key) name	            Type        Value
+=================================== =========== ==============
+``count``                           integer     How many times this additional field appears
+``examples``                        array*      An array of values for this field
```

Comment:
What is the maximum length of this array?

```suggestion
``examples``                        array*      A sample of values of the field
```

Reply:
3. I'll add this.